_sections/30-bindings.md

```diff
@@ -112,8 +112,8 @@ A subset of [XPath 1.0 functions](http://www.w3.org/TR/xpath/#corelib), some fun
 <a id="fn:boolean-from-string" href="#fn:boolean-from-string">`boolean-from-string(string arg)`</a> | Returns true if arg is "true" or "1", otherwise returns false.
 <a id="fn:not" href="#fn:not">`not(boolean arg)`</a> | As in [XPath 1.0](http://www.w3.org/TR/xpath/#function-not).
 <a id="fn:number" href="#fn:number">`number(* arg)`</a> | As in [XPath 1.0](http://www.w3.org/TR/xpath/#function-number).
-<a id="fn:decimal-date-time" href="#fn:decimal-date-time">`decimal-date-time(dateTime value)`</a> | Converts dateTime value to a number.
-<a id="fn:decimal-time" href="#fn:decimal-time">`decimal-time(time value)`</a> | Converts time value to a number.
+<a id="fn:decimal-date-time" href="#fn:decimal-date-time">`decimal-date-time(dateTime value)`</a> | Converts dateTime value to a number in the format used by Excel. The number before the decimal marker is the number of days since January 1, 1970 UTC. The number after the decimal marker represents the time as a fractional day.
```

Comment:
Hi @lognaturel, I couldn't find your earlier comment to respond to. What I meant was, the part about after the decimal marker seems confusing especially if we take into account timezone conversion. Would something like this be better perhaps (there is no separate before-decimal-marker and after-decimal-marker calculation I think):

`Converts dateTime value to the number of days since January 1, 1970 UTC`

Reply:
That works for me. `decimal-time` explains the fractional day idea if it's unclear. I'll still include the fact that it matches the Excel format because I think it helps explain why this was picked over more traditional epoch representations.

For old comments, there's a little "Show outdated" toggle somewhere up there ⏫ . 😊

Reply:
Ah! Thanks. I see the old comments now!

Sorry, I was not aware of any other decimal epoch representations (apart from using seconds or milliseconds instead of days). So maybe mentioning Excel makes sense if there is scope for confusion.

Reply:
Seconds or milliseconds is what I know. I hadn't seen the fractional day one before but that doesn't mean it's not common! When you wrote "Wow. I would have never ever guessed this! The number after the decimal marker means 3:00am is x.25 and noon is x.50?" up there ⏫ I thought you might have been surprised by it too!

Reply:
:) Yes, I was initially thrown off by the description, thinking there was a different meaning of the value before and after the decimal marker. But then realized there isn't. So it's just the unit that differs.